include/envoy/network/connection.h

```diff
@@ -86,6 +88,8 @@ class Connection : public Event::DeferredDeletable, public FilterManager {
     Stats::Gauge& write_current_;
     // Counter* as this is an optional counter. Bind errors will not be tracked if this is nullptr.
     Stats::Counter* bind_errors_;
+    // Optional counter. Delayed close semantics are only used by HTTP connections.
+    Stats::Counter* delayed_close_timeouts_;
```

Comment:
Agreed the delayed close timeout is not inherently HTTP specific, but the intent of the comment is to point out that _as currently implemented_ it only affects HTTP connections, since the H{1,2} codecs are the only callers of `setDelayedCloseTimeout()` on a connection to enable it.

I'm fine removing this however if you don't think the note/clarification is required.

Reply:
My feeling is that it's just a comment we'll have to delete (or will be incorrect) when someone inevitably enables this for a different purpose.

Reply:
Ok. Cleaned up the comment.